modules/ocf_mirrors/files/healthcheck

```diff
@@ -65,6 +82,18 @@ def get_updated_debian(mirror_url):
     return dateutil.parser.parse(updated_line.split(': ', 1)[1])
 
 
+def get_updated_manjaro(mirror_url):
+    """Find the time the host was last synced.
+    The last line on the page should be of the form "date=2018-01-25T21:52:18Z"
```

Comment:
Is this explicitly specified? If not I would go for regex matching lest they decide to append more lines later.